src/main/java/de/chojo/repbot/commands/Channel.java

```diff
@@ -70,56 +68,93 @@ private void whitelist(SlashCommandInteractionEvent event, SlashCommandContext c
+                            Replacement.create("CHANNEL", channel.getAsMention()))).queue();
+        }
     }
 
     private void remove(SlashCommandInteractionEvent event, SlashCommandContext context, Channels channels) {
-        var channel = event.getOption("channel").getAsMessageChannel();
-        if (channel == null || channel.getType() != ChannelType.TEXT) {
-            event.reply(context.localize("error.invalidChannel")).setEphemeral(true).queue();
+        var channelType = event.getOption("channel").getChannelType();
+        if (channelType != ChannelType.TEXT && channelType != ChannelType.CATEGORY) {
+            event.reply(context.localize("error.onlyTextOrCategory")).setEphemeral(true).queue();
             return;
         }
-        channels.remove(channel);
-
-        event.reply(context.localize("command.channel.sub.remove.removed",
-                Replacement.create("CHANNEL", channel.getAsMention()))).queue();
+        if (channelType == ChannelType.TEXT) {
+            var channel = event.getOption("channel").getAsTextChannel();
+            channels.remove(channel);
+            event.reply(
+                    context.localize("command.channel.sub.remove.removed",
+                            Replacement.create("CHANNEL", channel.getAsMention()))).queue();
+        } else {
+            var channel = event.getOption("channel").getAsGuildChannel();
+            channels.remove((Category) channel);
+            event.reply(
+                    context.localize("command.channel.sub.remove.removed",
+                            Replacement.create("CHANNEL", channel.getAsMention()))).queue();
+        }
     }
 
     private void list(SlashCommandInteractionEvent event, SlashCommandContext context, Channels channels) {
-        event.reply(getChannelList(channels, context)).queue();
+        event.replyEmbeds(getChannelList(channels, context)).queue();
     }
 
-    private String getChannelList(Channels channels, SlashCommandContext context) {
-        var channelNames = channels.channels().stream().map(IMentionable::getAsMention).collect(Collectors.joining(", "));
-        var message = "command.channel.sub.list." + (channels.isWhitelist() ? "whitelist" : "blacklist");
-        return context.localize(message, Replacement.create("CHANNEL", channelNames));
+    private MessageEmbed getChannelList(Channels channels, SlashCommandContext context) {
+
+        var channelNames = channels.channels().stream().map(IMentionable::getAsMention).limit(40).collect(Collectors.joining(", "));
+        if (channels.channels().size() > 40) {
+            channelNames += String.format("$%s$", "command.channel.list.more");
+        }
+
+        var categoryNames = channels.categories().stream().map(IMentionable::getAsMention).limit(40).collect(Collectors.joining(", "));
+        if (channels.categories().size() > 40) {
+            categoryNames += String.format("$%s$", "command.channel.list.more");
```

Comment:
```suggestion
            categoryNames += "$command.channel.list.more$";
```

Reply:
I used the format in order for IDEs to recognize this locale code as used. Otherwise it cant be found by usage anymore. We could store this as a final variable to not execute the format every time.